google-cloud-spanner/src/main/java/com/google/cloud/spanner/Value.java

```diff
@@ -218,7 +224,82 @@ public static Value pgJsonb(@Nullable String v) {
   }
 
   /**
-   * Returns a {@code BYTES} value.
+   * Return a {@code PROTO} value.
+   *
+   * @param v Not null Proto message.
+   */
+  public static Value protoMessage(AbstractMessage v) {
+    Preconditions.checkNotNull(v, "Use protoMessage(ByteArray, Descriptor) for null values.");
+    return protoMessage(
+        ByteArray.copyFrom(v.toByteArray()), v.getDescriptorForType().getFullName());
+  }
+
+  /**
+   * Return a {@code PROTO} value
+   *
+   * @param v Serialized Proto Array, which may be null.
+   * @param protoTypFqn Fully qualified name of proto representing the proto definition.
```

Comment:
I think it would be good to add an example how a user can easily get that (e.g. `MyProtoClass.getDefaultInstance().getFqnName()` or something like that?)